provider/spark.go

```diff
@@ -911,6 +998,62 @@ func (e *EMRExecutor) SparkSubmitArgs(destPath string, cleanQuery string, source
 	return argList
 }
 
+func createLogS3FileStore(emrRegion string, s3LogLocation string, awsAccessKeyId string, awsSecretKey string) (FileStore, error) {
```

Comment:
Can we tie these functions to something so theyre not free floating? Unless they're going to be used somewhere else

Reply:
This one is difficult to it down to something because it is used in `NewEMRExecutor` and it is really related to anything.

Reply:
got it